### PR TITLE
Update tor binary to 0.4.7.10

### PR DESCRIPTION
**Pull request by napoly**
not aiming for https://github.com/haveno-dex/haveno/issues/789 here.. rather just updating Bisq NetLayer to the latest possible.. hoping to resolve arm64 Ubuntu issue along the way

### CODE REVIEW

**Comment by woodser** [comment]
Getting this error on Ubuntu arm64:

``
Mar-10 20:47:41.940 [StartTor] ERROR h.n.p2p.network.TorNetworkNode: Starting tor node failed org.berndpruenster.netlayer.tor.TorCtlException: Could not setup Tor
	at org.berndpruenster.netlayer.tor.NativeTor.<init>(NativeTor.kt:106)
	at org.berndpruenster.netlayer.tor.NativeTor.<init>(NativeTor.kt:55)
	at org.berndpruenster.netlayer.tor.NativeTor.<init>(NativeTor.kt)
	at haveno.network.p2p.network.NewTor.getTor(NewTor.java:107)
	at haveno.network.p2p.network.TorNetworkNode.lambda$createTorAndHiddenService$10(TorNetworkNode.java:169)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.io.IOException: java.io.IOException: Tor exited with value 135
	at org.berndpruenster.netlayer.tor.TorContext.installAndStartTorOp(TorContext.kt:392)
	at org.berndpruenster.netlayer.tor.NativeTor.<init>(NativeTor.kt:66)
	... 8 common frames omitted
Caused by: java.io.IOException: Tor exited with value 135
	at org.berndpruenster.netlayer.tor.TorContext.installAndStartTorOp(TorContext.kt:354)
	... 9 common frames omitted

org.berndpruenster.netlayer.tor.TorCtlException: Could not setup Tor
	at org.berndpruenster.netlayer.tor.NativeTor.<init>(NativeTor.kt:106)
	at org.berndpruenster.netlayer.tor.NativeTor.<init>(NativeTor.kt:55)
	at org.berndpruenster.netlayer.tor.NativeTor.<init>(NativeTor.kt)
	at haveno.network.p2p.network.NewTor.getTor(NewTor.java:107)
	at haveno.network.p2p.network.TorNetworkNode.lambda$createTorAndHiddenService$10(TorNetworkNode.java:169)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.io.IOException: java.io.IOException: Tor exited with value 135
	at org.berndpruenster.netlayer.tor.TorContext.installAndStartTorOp(TorContext.kt:392)
	at org.berndpruenster.netlayer.tor.NativeTor.<init>(NativeTor.kt:66)
	... 8 common frames omitted
Caused by: java.io.IOException: Tor exited with value 135
	at org.berndpruenster.netlayer.tor.TorContext.installAndStartTorOp(TorContext.kt:354)
	... 9 common frames omitted
Mar-10 20:47:41.941 [JavaFX Application Thread] ERROR h.core.app.P2PNetworkSetup: onSetupFailed
```

**Comment by napoly** [comment]
the root cause of the issue seems to be that `Tor exited with value 135` during its setup process which generally indicates some form of a runtime failure.. still worth to merge as this is the most actual / tested by Bisq community I believe

File: common/src/main/java/haveno/common/config/Config.java [hunk 1]
```diff
@@ -446,6 +448,11 @@ public Config(String defaultAppName, File defaultUserDataDir, String... args) {
                         .withValuesConvertedBy(RegexMatcher.regex("^([^\\s,]+\\s[^,]+,?\\s*)+$"))
                         .defaultsTo("");
 
+        ArgumentAcceptingOptionSpec<String> torControlHostOpt =
+                parser.accepts(TOR_CONTROL_HOST, "The control hostname of an already running Tor service to be used by Bisq.")
```
Comment (woodser) [inline common/src/main/java/haveno/common/config/Config.java:452]: Should be Haveno.

**Comment by woodser** [comment]
Thanks very much.

**Pull request merged**